src/corona_hakab_model/detection_model/healthcare.py

```diff
@@ -28,14 +28,24 @@ class PendingTestResults(Queue[PendingTestResult]):
 
 
 class DetectionTest:
-    def __init__(self, state_to_detection_prop: Dict[MedicalState, float], time_until_result):
+    def __init__(self,
+                 state_to_detection_prop: Dict[MedicalState, float],
+                 time_dist_until_result):
         self.state_to_detection_prop = deepcopy(state_to_detection_prop)
-        self.time_until_result = time_until_result
+        self.time_dist_until_result = time_dist_until_result
+
+    def get_times_to_get_results(self, number_of_agents=None):
```

Comment:
why not set number_of_agents to 1 as default, and get rid of the if statement?

Reply:
I now rememeber why I did this. This function should return a number when size=1,  but if you give choice 1 it return an ndarray, so if I give 1 as input I call it without size=1 (look at the implementation)